### DISCUSSION TITLE
Cibercriminosa

### DISCUSSION THREAD

**Discussion by Jofernmorais** [community/community#83052] [Discussions]
Bom dia,
Tenho sofrido constantes ataques em minha conta Github Jofernmorais, por cibercriminos Brasil São Paulo. Eles estão constantemente invadindo a minha conta utilizando outro perfil Jofermorais. Gostaria que essa empresa monitorasse as atividades e adotassem providências no sentido de coibir tal prática.

Atenciosamente
JOSÉ FERNANDES MORAIS

**Comment by shanestover8991** [comment]
Gracias José, que tengo búon dia!

**Comment by rajan2133** [comment]
#83055

**Comment by rajan2133** [comment]
#54848

**Comment by shanestover8991** [comment]
[https://www.freecodecamp.org/espanol/news/python-ejemplos-de-codigo-tutorial-de-programacion-en-python-desde-cero-para-principiantes/](https://www.freecodecamp.org/espanol/news/python-ejemplos-de-codigo-tutorial-de-programacion-en-python-desde-cero-para-principiantes/)

**Comment by Akash1134** [comment]
Hi @Jofernmorais ,

Please note that we take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. **Unfortunately, we don’t currently have moderators for languages other than English**. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

However, considering your report, we really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
